### PR TITLE
Add section for configuring scim for roles

### PR DESCRIPTION
**Pull request by AniketDas-Tekky**
This section details how to set up org role provisioning for Okta

### CODE REVIEW

**Review by leedongwei — CHANGES_REQUESTED** [review]
I've sent screenshots through Slack, please add them in.

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 1]
```diff
@@ -63,6 +63,46 @@ As a result of these changes, users who are assigned will be sent an invitation
...
+
+</Alert>
+
+You can assign an organization-level role to an okta group
+
+1. Add a new custom attribute to your Okta application profile
+
+a. Navigate your your application settings in Okta
+
+b. Under the "Provisioning" tab, select "Go to Profile Editor"
```
Comment (cathyteng17) [inline src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:80]:
```suggestion
b. Under the "Provisioning" tab under the "Attribute Mappings" header, select "Go to Profile Editor". If you do not see this, please configure your app for provisioning to Sentry first.
```

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 2]
```diff
@@ -63,6 +63,46 @@ As a result of these changes, users who are assigned will be sent an invitation
...
+d. Fill out the form with the following settings (You can set whatever value you want for any setting not listed below)
+
+    - Data Type: `string`
+    - External name: `sentryOrgRole`
+    - External namespace: `urn:ietf:params:scim:schemas:core:2.0:User`
+    - Attribute type: `Group`
+
+2. Assign a group to your okta application
+
+3. In the form, type out the role you want every member of the group to be.
```
[the remark this answers is not in the document]
Comment (cathyteng17) [reply src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:93]: will we be specifying here that single tenant instances may have their own roles?

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx (hunk 1 above)
Comment (cathyteng17) [reply src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:80]: Screen Shot 2023-01-06 at 09 50 27
can we also mention that you need this checked?

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (cathyteng17) [reply src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:93]: can we also mention that simply assigning the role in okta will provision the sentry org member with the role? folks won't need to push the group unless they want a team with those users

**Review by cathyteng17 — COMMENTED** [review]
Screen Shot 2023-01-06 at 09 26 42
should we mention somewhere that you should have "Enable for SCIM" checked?

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 3]
```diff
@@ -63,6 +63,46 @@ As a result of these changes, users who are assigned will be sent an invitation
...
+
+- Group members will be provisioned with the organization default role if the field is left blank
+- Invalid role names will prevent group members from being provisioned. To try again, you'll need to remove the group first.
+- A list of valid roles can be found by going to the any organization member's settings page.
+- For security, roles with the `org:admin` permission scope cannot my provisioned through SCIM.
+
+<Alert level="note">
+
+If a user is a member of multiple okta groups, only the attributes set by the first group will be used for the user.
```
[the remark this answers is not in the document]
Comment (AniketDas-Tekky) [reply src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:102]: The page is specifically for okta. Should I still be more generic?

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx (hunk 1 above)
Comment (AniketDas-Tekky) [reply src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:80]: So the section for role provisioning lives under general Okta SCIM setup page. The page already mentions to `Enable SCIM` and to enable creating users, so I don't think we need to mention it again here.

Comment (AniketDas-Tekky) [reply src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:80]: See https://docs.sentry.io/product/accounts/sso/okta-sso/okta-scim/#sentry-configuration

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 4]
```diff
@@ -63,6 +64,50 @@ As a result of these changes, users who are assigned will be sent an invitation
 
 You can use "Push Groups" to sync and assign groups in Okta; they will be reflected in Sentry teams.
 
+### Configuring organization-level roles (Beta)
+
+<Alert level="note">
+
+Organization role provisioning is in beta. If you wish to try this feature out, contact support@sentry.io
```
Comment (lizokm) [inline src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:71]:
```suggestion
Organization role provisioning is in beta. If you'd like to try this feature, contact support@sentry.io.
```

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 5]
```diff
@@ -63,6 +64,50 @@ As a result of these changes, users who are assigned will be sent an invitation
...
+
+</Alert>
+
+You can assign an organization-level role to an okta group
```
Comment (lizokm) [inline src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:75]:
```suggestion
Here's how to assign an organization-level role to an Okta group:
```

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 6]
```diff
@@ -63,6 +64,50 @@ As a result of these changes, users who are assigned will be sent an invitation
...
+
+  c. Select "+ Add Attribute"
+
+  d. Fill out the form with the following settings (You can set whatever value you want for any setting not listed below)
+
+    * Data Type: `string`
+    * External name: `sentryOrgRole`
+    * External namespace: `urn:ietf:params:scim:schemas:core:2.0:User`
+    * Attribute type: `Group`
+  ![Okta Custom Attribute](./okta-custom-attribute-values.png)
```
Comment (lizokm) [inline src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:92]:
```suggestion
1. Add a new custom attribute to your Okta application profile.

  a. Navigate to your application settings in Okta

  b. Under the "Provisioning" tab, select "Go to Profile Editor"
        ![Okta Provisioning Page](./okta-go-to-profile.png)

  c. Select "+ Add Attribute"

  d. Fill out the form with the following settings (you can set whatever value you want for any setting not listed below):

    * Data Type: `string`
    * External name: `sentryOrgRole`
    * External namespace: `urn:ietf:params:scim:schemas:core:2.0:User`
    * Attribute type: `Group`
  ![Okta Custom Attribute](./okta-custom-attribute-values.png)
```

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 7]
```diff
@@ -63,6 +64,50 @@ As a result of these changes, users who are assigned will be sent an invitation
...
+
+2. Assign a group to your okta application
```
Comment (lizokm) [inline src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:94]:
```suggestion
2. Assign a group to your Okta application.
```

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 8]
```diff
@@ -22,6 +22,7 @@ If you change your organization slug, you'll need to make the same update in the
 - Deactivate users
 - Push groups
 - Import groups
+- Configure organization-level role
```
Comment (lizokm) [inline src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:25]:
```suggestion
- Configure organization-level roles
```

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 9]
```diff
@@ -63,6 +64,50 @@ As a result of these changes, users who are assigned will be sent an invitation
...
+2. Assign a group to your okta application
+    ![Okta Assign Group](./okta-assign-group.png)
+
+3. In the form, enter the string for the org-level role.
+    ![Okta Set Role](./okta-set-group-attribute.png)
+
+- If the `sentryOrgField` field is left blank, Group members will be provisioned with the default organization-level role. This default role is configured in Sentry, under Settings -> Organization -> Auth
+- Invalid role names will prevent group members from being provisioned. To try again, you'll need to remove the group first.
+- A list of valid roles can be found by going to the any organization member's settings page.
+- For security, roles with the `org:admin` permission scope cannot my provisioned through SCIM.
```
Comment (lizokm) [inline src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:103]:
```suggestion
- If the `sentryOrgField` field is left blank, group members will be provisioned with the default organization-level role. This default role is configured in Sentry, under Settings -> Organization -> Auth
- Invalid role names will prevent group members from being provisioned. To try again, you'll need to remove the group first.
- A list of valid roles can be found in the organization member's settings page.
- For security, roles with the `org:admin` permission scope cannot be provisioned through SCIM.
```

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 10]
```diff
@@ -63,6 +64,50 @@ As a result of these changes, users who are assigned will be sent an invitation
...
+
+<Alert level="note">
+
+If a user is a member of multiple okta groups, only the attributes set by the first group will be used for the user.
```
Comment (lizokm) [inline src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:107]:
```suggestion
If a user is a member of multiple Okta groups, only the attributes set by the first group will be used.
```

**Review by lizokm — APPROVED** [review]
Made some small tweaks.

**Review by cathyteng17 — APPROVED** [review]

**Pull request merged**